docs/nuget-org/policies/deleting-packages.md

```diff
@@ -34,3 +34,27 @@ Packages that meet any of the following criteria are not allowed on the public N
 If you find a package that is in violation of any of these items, click the **Report Abuse** link on the package details page and submit a report.
 
 Note that the NuGet team and the .NET Foundation reserves the right to change these criteria at any time.
+
+#### Unlisting a package
```

Comment:
Perhaps this should be a "higher level" heading like `##`? With this, it looks like it's a subsection of "Prohibited use".